### DISCUSSION TITLE
Progress Forum Minimal Feature Set

### DISCUSSION THREAD

**Discussion by alecfwilson** [progressforum/ForumMagnum#4] [General]
I took a look at LessWrong (https://www.lesswrong.com/) and found what I consider to be the minimal feature set for the Progress Forum. The goal I am operating under is that we should use as much of what LW has built as possible while keeping in mind that we need to minimize the net new content needed prior to launch. Also, I am focusing on what the forum users will see. If there are technical issues with implementing anything as is from LW, please comment and we can adjust.

My recommendation:
* Keep the homepage as is, except in the "Recommendations" section that includes 3 Sequences for LessWrong, we should probably just have a "Start Here" section
* Limit the nav to Library, Community, All Posts, Subscribe, Contact, About, and FAQ. Library should initially be labeled "Start Here" and link directly to a Start Here sequence, unless we're able to develop a few more sequences before launch
* Implement the Community page as is, except perhaps disabling the cross-listing and affiliated orgs
* Implement the All Posts page as is
* Build a Concepts page that is initially hidden from the nav, and start with 6 high level tags. Plan to expand this aggressively post launch
* Build support Sequences on the Library page as LessWrong has implemented, plan to aggressively solicit sequences to populate the page with
* Plan to support both Open Questions and Shortform at launch, disucss whether or not they should be linked in the Nav at launch. Build both as LW has implemented.
* Implement accounts, login, search, etc as similarly to LW as possible.

My in-progress notes on LW's structure can be found [here](https://github.com/Progress-Studies-Forum/ForumMagnum/wiki/afw-lw-notes)

**Comment by jasoncrawford** [comment]
This sounds good, thanks @alecfwilson. If we need intro content we can definitely come up with a reading list that would include, for instance, Tyler and Patrick's original Atlantic article, a post or two from The Roots of Progress, etc.

**Reply by michaelmagoon** [reply]
I think keeping MVP as simple as possible makes a lot of sense. I will defer to Alec as to the best strategy for MVP. My biggest concern is what happens after MVP. I think assessing the functionality and extensibility of ForceMagnum compared to my "long-term vision" design is a high priority for this week/weekend. It should be pretty easy.
As to Jason's point, I also have a lot of relevant links to progress-related content on my two websites. I have no desire to monopolize the content, but I also do not want to be left out either. I assume that copy/pasting in hyperlinks is easy, correct?

I created and maintained a list of progress-related content for the past few months. I have been hosting these links on my site because there was nowhere else to put them. I am glad to have a better location for it. 
Based upon what you said above, I think they belong on the "Community" page:

This is a list of all progress-related online resources that I know. Please copy/paste all links on this page to the "Community" page:
- https://frompovertytoprogress.com/resources/

If we are creating a list of important progress-related posts, please add these two from my blog to that list:
- https://frompovertytoprogress.com/2021/04/06/what-are-the-causes-of-progress-magoon/
- https://frompovertytoprogress.com/2021/12/08/the-five-keys-to-progress/

**Reply by michaelmagoon** [reply]
Oops, sorry. I thought that I was replying to Alec, not to Jason....

**Reply by jasoncrawford** [reply]
Thanks Michael. This is a good list of links and we'll consult it (along with similar lists) when we're putting together any content for the site.

**Comment by michaelmagoon** [comment]
Unless I misunderstood or someone objects, I believe that we agreed on Slack to Andrew Robert's modification of my original design for MVP.

**Reply by jasoncrawford** [reply]
I think there's been a misunderstanding; see my reply in Slack. Let's use Alec's doc above as the spec and keep it updated with any decisions.
